src/ui/src/app/chart-search/chart-search.component.ts

```diff
@@ -1,56 +0,0 @@
-import { Component, OnInit } from '@angular/core';
```

Comment:
You can still link directly to a search with `/charts?q=test`
And having an endpoint where you only have the result and do nothing else was kind of painful. We remove a route for simplicity, and the user is back on a page he already knows (charts-list) where he can filter/sort and keep searching